src/fragments/lib/push-notifications/react-native/getting_started/60_init_push_notifications.mdx

```diff
@@ -1,41 +1,41 @@
 To initialize Amplify Push Notifications, you will first need to configure Amplify.
 
-Import and load the configuration file in your app. It's recommended you add the Amplify configuration step to your app's root entry point. For example, `index.js`
+Import and load the configuration file in your app. It's recommended you add the Amplify configuration step to your app's root entry point if your app requires push notification handling even while in a terminated state. For example, `index.js`
```

Comment:
This change is true for initializePushNotifications API explanation in the next block right? Not here.

Reply:
To init in the root, you'll have to first config in the root

Reply:
Yeah but `Amplify.configure` doesn't really setup the push notification handling in the terminated state, it's the function of `initializePushNotifications` API so it might be confusing to them where they think just doing `Amplify.configure` is enough